### ISSUE TITLE
SQL Error: Internal jdbc driver error (Data Tab)

### ISSUE THREAD

**Issue by Slider1156** [dbeaver/dbeaver#2679]
I connected DBeaver to Oracle 11g XE (additional .jar files: orai18n.jar, xdb6.jar and ojdbc6.jar). But when I try to open Data tab to fill tables with data i get an SQL Error: Internal jdbc driver error. After researching I found out that I should add xmlparserv2.jar in driver manager but it seems that there is no xmlparserv2.jar for oracle 11g. IMO It is basically saying that DBeaver cannot store data anywhere even though it is connected to Oracle 11g and is given 50M of space.

Error Message
![](https://imgur.com/ssa88xg.jpg)

Driver Manager
![](https://imgur.com/DG1G8HY.jpg)

This is how my Data tab looks like when I close error window. Btw I also increased amount of memory available to DBeaver (-vmargs -Xmx2048M) but that didn't change anything.
![](https://imgur.com/KqbfPPp.jpg)

**Comment by serge-rider** [comment]
Perhaps you need to use Oracle 12g driver (like in #2629).
Also you don't need to extra jar files of Oracle driver. Just download them and add in DBeaver as jar files (not as directory).

Also you could try to download DBeaver EE (trial version) and check how it works - Oracle driver is included in distribution. If it works ok then you can just use the same driver configuration in CE.

**Comment by Slider1156** [comment]
DBeaver EE works perfectly! It's probably because it has included xmlparserv2.jar which I didn't have (other drivers where the same, except xdb6.jar that was replaced with xdb.jar). Wouldn't it be easier if xmlparserv2.jar is uploaded and available for download to those who need it so they don't have to download Enterprise Edition just for one .jar file?

I added .jar files instead of folders and included xmlparserv2.jar in CE and it works! Thanks! :D

Btw I didn't try with Oracle 12g driver so I don't know if that works to anyone who is interested :)
